libraries/SITL/SITL.cpp

```diff
@@ -111,6 +117,10 @@ const AP_Param::GroupInfo SIM::var_info[] = {
     // @Units: V
     // @User: Advanced
     AP_GROUPINFO("BATT_VOLTAGE",  19, SIM,  batt_voltage,  12.6f),
+    // @Param: BATT_CAP_AH
+    // @DisplayName: Simulated battery capacity
+    // @Description: Simulated battery capacity in Amp hours
```

Comment:
```suggestion
    // @Description: Simulated battery capacity
    // @Units: Ah
```

Will need to add a unit to the above table of known units

Reply:
`Ah` added  as known unit